### DISCUSSION TITLE
php artisan chimera:delete

### DISCUSSION THREAD

**Discussion by workualemu** [tech-acs/chimera-starter-kit#44] [General]
Can we delete selected indicators, scorecards, reports or mapindicators?

**Comment by amestsantim** [comment]
Yes, that's is exactly what the chimera:delete command does!
First it will ask you what type of component you want to delete (indicator, scorecard...)
Then it will give you a dynamic menu of existing components of the type you have selected... then it will delete both the database entry and the file.

**Comment by amestsantim** [comment]
Also, rather than per repository, we host all discussions on the main discussion board...

https://github.com/orgs/tech-acs/discussions

**Comment by infobcr** [comment]
I want to know how to generate python to laravel application development
